.gitignore

```diff
@@ -1 +1,2 @@
 /config.h
+/alfred/build/**
```

Comment:
this is not mqtt related

Reply:
Correct - more convinience that absolutely necessary.